### ISSUE TITLE
OpenAI-GPT3 search endpoint deprecated 

### ISSUE THREAD

**Issue by rajarajanvakil** [Muennighoff/sgpt#5]
Hello there, the search end point(purpose) as been deprecated is there any alternative solution on this. This happens at CE-BEIR notebook. 
Thanks in advance.

**Comment by Muennighoff** [comment]
Hey are you reffering to the `crossencoder_openai.ipynb` notebook?

You can still use the OpenAI search endpoint using the workaround in [this script](https://github.com/Muennighoff/sgpt/blob/main/crossencoder/beir/openai_search_endpoint_functionality.py) using their completion endpoint.

**Comment by GouthamVicky** [comment]
Hello , Even if we try to run this [script](https://github.com/Muennighoff/sgpt/blob/main/crossencoder/beir/openai_search_endpoint_functionality.py) following error occurs 

`openai.error.InvalidRequestError: You do not have access to the search endpoint, likely because it is deprecated. Please see https://community.openai.com/t/answers-classification-search-endpoint-deprecation/18532 for more information and reach out to deprecation@openai.com if you have any questions.`

- [ ] Python version - 3.9
- [ ] Open AI pip version - 0.20.0

Thanks

**Comment by Muennighoff** [comment]
Yeah you probably need to remove the `print(openai.Search.create(model="davinci", query=query, documents=docs))` part & the BM25 is missing I think.

It may be cheaper for you to just use the Cross-Encoder method we propose in this repo [here](https://github.com/Muennighoff/sgpt#asymmetric-semantic-search-1).
Or use a Bi-Encoder with embeddings.

**Comment by GouthamVicky** [comment]
yeah it's working fine now, Thanks for the quick response
